### PR TITLE
Add admin interface

### PR DESCRIPTION
**Pull request by casassg**
Admin interface added to project

Features included:
- Invite applications (#5)
- Accept applications (#5)
- Review applications (#5)
- Add new users (#4)
- Remove users (#4)

Other features:
- Typeform full serialization (#2)

### CODE REVIEW

**Comment by casassg** [comment]
closes #4 and #5

File: register/models.py [hunk 1]
```diff
@@ -73,6 +100,9 @@ def confirmation_url(self, request=None):
     def cancelation_url(self, request=None):
         return reverse('cancel_app', kwargs={'token': self.id}, request=request)
 
+    def test_url(self, request=None):
```
Comment (ani300) [inline register/models.py]: ???

File: register/models.py [hunk 2]
```diff
@@ -39,18 +43,41 @@ class Application(models.Model):
...
     # Needs to be set to true -> else rejected
     authorized_mlh = models.NullBooleanField()
-    status = models.CharField(choices=status, default='P', max_length=1)
+    status = models.CharField(choices=status, default='P', max_length=2)
+
+    @property
+    def positive_votes(self):
+        return self.vote_set.filter(vote=1).count()
+
+    @property
```
Comment (ani300) [inline register/models.py]: Maybe here return the *3 number, so ppl don't get confused between total and so on

File: register/forms.py [hunk 3]
```diff
@@ -48,11 +48,58 @@ def save(self, application):
         application.save()
 
     def map(self, obj):
```
Comment (ani300) [inline register/forms.py]: Using a name for a function which already exists in the language?

File: register/forms.py [hunk 4]
```diff
@@ -23,22 +20,25 @@ def save(self, application):
...
-        return map(self.save, applications)
+        return [app.save() for app in applications]
 
 
 class TypeformFetcher(ApplicationFormFetcher):
     base_url = 'https://api.typeform.com/v1/form/'
 
-    def get_form_id(self):
-        return settings.REGISTER_APP['applications_form_id']
+    @property
```
Comment (ani300) [inline register/forms.py:45]: Do this and all the "Not implemented" functions need to be defined? If so, are they not being used?

**Review by ani300 — CHANGES_REQUESTED** [review]
I have a few comments/questions on the code.
I have guessed the migration part is OK and that the application is currently working.
I have only looked at how the code was written.

File: register/models.py (hunk 1 above)
Comment (casassg) [reply register/models.py]: True, removing this...

File: register/forms.py (hunk 4 above)
Comment (casassg) [reply register/forms.py:45]: It's an option to define abstract methods in a class. If the method is called gives NotImplemented. Following code style from Django Rest Framework. 

From: https://docs.python.org/2/library/exceptions.html

`In user defined base classes, abstract methods should raise this exception when they require derived classes to override the method.`

File: register/forms.py (hunk 3 above)
Comment (casassg) [reply register/forms.py]: True, renaming

**Review by ani300 — APPROVED** [review]

File: register/models.py (hunk 2 above)
Comment (casassg) [reply register/models.py]: `positive_votes` and `negative_votes` properties are not being used, so they can be removed. set both inside votes properties

**Comment by casassg** [comment]
Merging...

**Pull request merged**